src/navigation/wallet/screens/TransactionProposalDetails.tsx

```diff
@@ -231,6 +228,10 @@ const TransactionProposalDetails = () => {
         }),
       );
       setTxs(_transaction);
+      setLastSigner(
+        _.filter(_transaction.actions, {type: 'accept'}).length ===
```

Comment:
lodash is not necessary here, you can use the `filter` method corresponding to javascript arrays ```_transaction.actions.filter(action => ...```